### PR TITLE
Gabi code upgrades

### PR DESCRIPTION
**Pull request by wissog**
removed that windows-build-tool and added more stuff

### CODE REVIEW

File: src/components/Sections/Library/LibraryContentContainer.js [hunk 1]
```diff
@@ -46,133 +29,117 @@ const getFullUrl = (pdf, data, language, source) => {
...
+    history,
+    langSelectorMount = null, 
+    downloadAllowed } = props;
+
+  const [language, setLanguage] = useState(null);
+  const [languages, setLanguages] = useState([]);
+
+  useEffect(() => {
+    if (data){
+      const languages =[...Object.keys(data)];
```
Comment (gshilin) [inline src/components/Sections/Library/LibraryContentContainer.js]: Isn't Object.keys(data) enough?

File: src/components/Sections/Publications/tabs/Blog/Post/Container.js [hunk 2]
```diff
@@ -1,83 +1,51 @@
...
+  const err = useSelector(state => selectors.getBlogErrorPost(state.publications));
+  const language = useSelector(state => settings.getLanguage(state.settings));
+ 
+  const { blog, id } = match.params;
+  const post = useSelector(state => selectors.getBlogPost(state.publications, blog, id));
+
+  const dispatch = useDispatch();
+
+  useEffect(
+    () => {
```
Comment (gshilin) [inline src/components/Sections/Publications/tabs/Blog/Post/Container.js]: Pls return conditions back

File: src/components/Sections/Library/LibraryContentContainer.js (hunk 1 above)
Comment (wissog) [reply src/components/Sections/Library/LibraryContentContainer.js]: yea

File: src/components/Sections/Publications/tabs/Blog/Post/Container.js (hunk 2 above)
Comment (wissog) [reply src/components/Sections/Publications/tabs/Blog/Post/Container.js]: done

**Pull request merged**